subsys/usb/usb_device.c

```diff
@@ -978,6 +975,73 @@ static bool is_ep_valid(uint8_t ep)
 	return false;
 }
 
+static bool usb_get_status_endpoint(struct usb_setup_packet *setup,
+				    int32_t *len, uint8_t **data_buf)
+{
+	uint8_t ep = setup->wIndex;
+	uint8_t *data = *data_buf;
+
+	/* Check if request addresses valid Endpoint */
+	if (!is_ep_valid(ep)) {
+		return false;
+	}
+
+	/** This request is valid for Control Endpoints when
```

Comment:
I would suggest changing `/**` to `/*` here just so it isn't picked up by doxygen.